### ISSUE TITLE
"Show thumbnails" is set to Never but I still get the "thumbnail problem" message bar

### ISSUE THREAD

**Issue by temporaryaccount** [linuxmint/nemo#1919]
```
 * Nemo 3.8.5
 * Mint Tara
 ```

**Issue**

I disabled the thumbnails from the Preferences window (Preview tab, Show thumbnails: Never), but the ~/.cache/thumbnails/normail directory was being filled with thumbnails nonetheless. I revoked the write permission from my user on that directory, but now Nemo show this annoying bar and doesn't remember my answer (Dismiss).

**Steps to reproduce**

$ rm -f ~/.cache/thumbnails/normal/*

$ chmod 000 ~/.cache/thumbnails/normal

open Nemo, it will show the annoying [bar](https://github.com/linuxmint/nemo/blob/master/src/nemo-thumbnail-problem-bar.c)

**Expected behaviour**

Either one of these:

* Don't even try to initialize the thumbnail generating code if the option was set to never show a thumbnail
* Show the problem message only in the "Preview" tab of the Preferences window
* Remember my choice to Dismiss the problem for good

Thanks.

**Comment by leigh123linux** [comment]
~/.cache/thumbnails/ is used by other apps other than nemo.
If I disable nemo it shows no thumbnails and doesn't create any.
I'm closing the issue as your steps to reproduce are complete gibberish and has nothing to do with the issue.

**Comment by leigh123linux** [comment]
@temporaryaccount 

You did!

> I disabled the thumbnails from the Preferences window (Preview tab, Show thumbnails: Never), but the ~/.cache/thumbnails/normail directory was being filled with thumbnails nonetheless.
As for this statement, I don't like you tone.
I know exactly what those command do, and why it would trigger your non-issue message bar.
> 
Ah, I see, it must be the first time you are seeing shell commands. OK then, goodbye.

**Comment by temporaryaccount** [comment]
> You did!

I was describing the reasons why I decided to revoke access permissions from the thumbnails cache directory: I disabled the thumbnails in Nemo but the directory was still being filled (maybe by other software), so as I couldn't find any other place in the system (or other software) to disable the generation of thumbnails I used `chmod 000 $dir`.

After that all comes my issue: Nemo is asked not to generate or show any thumbnails but it is complaining about the access rights on the thumbnails directory (which it shouldn't use). This is my issue, and I offered 3 different solutions in the "Expected behaviour" section.

> As for this statement, I don't like you tone.
> I know exactly what those command do, and why it would trigger your non-issue message bar.

I know that you known what those commands do (seeing your repositories and your work). My tone was triggered by your tone, it was a tit-for-tat snark after you called my two-line "Steps to reproduce" a `complete gibberish`. OK, the first line is not required to trigger the issue, it was what I did though (my exact steps) to reproduce the issue, why is that a "complete gibberish"? Anyway, I offer peace, if you think this is a non-issue then OK, no problem, thank you for what you do.
